examples/with_winit/src/lib.rs

```diff
@@ -462,10 +459,15 @@ impl ApplicationHandler<UserEvent> for VelloApp<'_> {
                 self.prior_position = None;
             }
             WindowEvent::CursorMoved { position, .. } => {
-                let position = Vec2::new(position.x, position.y);
+                let position = Point {
+                    x: position.x,
+                    y: position.y,
+                };
                 if self.mouse_down {
                     if let Some(prior) = self.prior_position {
-                        self.transform = Affine::translate(position - prior) * self.transform;
+                        self.transform = self
+                            .transform
+                            .then_translate(position.to_vec2() - prior.to_vec2());
```

Comment:
There is an `impl Sub<Point> for Point`, with output `Vec2`:

```suggestion
                        self.transform = self.transform.then_translate(position - prior);
```

Reply:
Thanks, will use that.